.pre-commit-config.yaml

```diff
@@ -1,137 +1,144 @@
+      - id: trailing-whitespace  # Remove trailing whitespace.
+      - id: name-tests-test  # Follow PyTest naming convention.
+
+  ########################################################################################
+  # Formatters: hooks that re-write Python & documentation files
+  ########################################################################################
+  # Convert relative imports to absolute imports
+  - repo: https://github.com/MarcoGorelli/absolufy-imports
+    rev: v0.3.1
+    hooks:
+      - id: absolufy-imports
+
+  # Make sure import statements are sorted uniformly.
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+
+  # Format the code
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        language_version: python3.10
+
+  # Remove f-string prefix when there's nothing in the string to format.
+  - repo: https://github.com/dannysepler/rm_unneeded_f_str
+    rev: v0.1.0
+    hooks:
+      - id: rm-unneeded-f-str
+
+  # Use built-in types for annotations as per PEP585
+  - repo: https://github.com/sondrelg/pep585-upgrade
+    rev: v1.0
+    hooks:
+      - id: upgrade-type-hints
+
+  # Update Python language constructs to modern standards
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.32.1
+    hooks:
+      - id: pyupgrade
+        args: [--py310-plus]
+
+  ########################################################################################
+  # Linters: hooks that check but don't alter Python and documentation files
+  ########################################################################################
+
+  # Check for PEP8 non-compliance, code complexity, style, errors, etc:
+  - repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        args: [--config, tox.ini]
+        additional_dependencies:
+          - flake8-docstrings
+          - flake8-colors
+          - pydocstyle
+          - flake8-builtins
+          - mccabe
+          - pep8-naming
+          - pycodestyle
+          - pyflakes
+          - flake8-rst-docstrings
+          - flake8-use-fstring
+
+  # Check for known security vulnerabilities:
+  - repo: https://github.com/PyCQA/bandit
+    rev: 1.7.4
+    hooks:
+      - id: bandit
+        args: [--configfile, .bandit.yml]
+
+  # Check for errors in restructuredtext (.rst) files under the doc hierarchy
+  - repo: https://github.com/PyCQA/doc8
+    rev: 0.11.2
+    hooks:
+      - id: doc8
+        args: [--config, tox.ini]
+
+  # Lint any RST files and embedded code blocks for syntax / formatting errors
+  - repo: https://github.com/myint/rstcheck
+    rev: v6.0.0.post1
+    hooks:
+      - id: rstcheck
+        additional_dependencies: [sphinx]
+        args: [--config, tox.ini]
+
+  # Format yaml files
+  - repo: https://github.com/lyz-code/yamlfix/
+    rev: master
+    hooks:
+      - id: yamlfix
```

Comment:
Should go in the formatter rather than linter section.

Also I think `pre-commit` wants specific revisions rather than dynamically changing branch refs like `master`.